doc/contribute/contributor_expectations.rst

```diff
@@ -133,6 +145,10 @@ PR Requirements
 - Changes to APIs must increment the API version number according to the API
   version rules.
 
+- Documentation must be added and/or updated to reflect the changes in the code
+  introduced by the PR. The documentation changes must use the proper
+  terminology as present in the existing pages, as well as be correctly written.
```

Comment:
again "correctly" is prob too vague.
also might be worth indicating that we use American English, I don't think this is mentioned anywhere.

Reply:
Agreed, fixed.